superset/viz.py

```diff
@@ -1907,7 +1936,7 @@ def get_df(self, query_obj: Optional[Dict[str, Any]] = None) -> pd.DataFrame:
         return pd.DataFrame()
 
     def get_data(self, df: pd.DataFrame) -> VizData:
-        return {}
```

Comment:
Is this a separate issue?

Reply:
Context was discussed in the meetup; adding here...

If data is null or empty on the client side we render the `no results` message. iframes should never have data, but should still render the base chart, so i'm adding some dummy data here to ensure iframe charts render properly